### PR TITLE
Minor UI improvements

### PR DESCRIPTION
**Pull request by roj1512**
- Made more areas of the homepage hero interactive (e.g., the header, the motto).
- Disabled selectablity in more areas that don't make sense.

### CODE REVIEW

**Comment by roj1512** [comment]
The PR is ready, but I would like to add some more improvements. Where is the source code for this located? I mean the installation commands' UI.

image

**Comment by roj1512** [comment]
Seems like that is in deno_doc. So marking this PR as ready for review.

File: frontend/islands/HomepageHero.tsx [hunk 1]
```diff
@@ -152,7 +152,7 @@ export function HomepageHero(
         <script src={asset("/scripts/particles.js")} defer></script>
       </Head>
       <style>{canvasStyle}</style>
-      <div class="section-x-inset-xl flex flex-col items-center justify-center gap-12 relative">
+      <div class="section-x-inset-xl flex flex-col items-center justify-center gap-12 relative pointer-events-none">
```
Comment (josh-collinsworth) [inline frontend/islands/HomepageHero.tsx:155]: I'm not so sure about this one. I understand disabling select for interactive controls (and it is nice to have the background interactivity here), but this is just a heading, and a user might legitimately want to select the text. I'd prefer to err on the side of trusting the user here.

**Review by josh-collinsworth — CHANGES_REQUESTED** [review]
Just one location where I feel these changes might be a little bit of an overreach. Otherwise, a solid addition.

File: frontend/islands/HomepageHero.tsx (hunk 1 above)
Comment (roj1512) [reply frontend/islands/HomepageHero.tsx:155]: This `pointer-events-none` is not for disabling selectability. Users can still select the text in the header. This lets the user interact with the animation (or whatever it is called) from empty parts of the header.

Comment (josh-collinsworth) [reply frontend/islands/HomepageHero.tsx:155]: Ah yes I see that; my mistake.

**Review by josh-collinsworth — APPROVED** [review]

**Pull request merged**